cime_config/machines/config_machines.xml

```diff
@@ -1269,7 +1268,7 @@
         <command name="load">acme-pfunit/3.2.8/base</command>
       </modules>
       <modules mpilib="!mpi-serial">
-        <command name="load">acme-openmpi/2.1.5</command>
+        <command name="load">acme-openmpi/4.1.4</command>
```

Comment:
Is this module even necessary, given that you prepend your OpenMPI installation to the PATH below?

Reply:
Oh, good catch. That was something I added before the sems/acme modules were available.